source/docs/hardware/hardware-safety/io_safety.rst

```diff
@@ -0,0 +1,27 @@
+Input/Output Safety Mechanisms Built into the 2015-2026 `FRC` Control System
```

Comment:
```suggestion
Input/Output Safety Mechanisms Built into the `FRC` Control System
```
calling out the years is overly specfici and not really helpful. If the next control system has different safety mechanisms, the document would be updated.